packages/jest-test-result/src/helpers.ts

```diff
@@ -145,3 +145,28 @@ export const addResult = (
     testResult.snapshot.unmatched +
     testResult.snapshot.updated;
 };
+
+export const emptyTestResult = (): TestResult => ({
+  leaks: false, // That's legacy code, just adding it so Flow is happy.
```

Comment:
we don't use flow 😀

Reply:
Copied the comment from the existing code. Should I update it as `Legacy code, just adding it as needed for typing`

Reply:
that sounds good, yeah 🙂